### ISSUE TITLE
Race condition causing egz.hope corruption

### ISSUE THREAD

**Issue by mikolajpp** [urbit/urbit#901]
I managed to corrupt my planet.
I was pasting some stuff in talk, and slow as it is I pressed ctrl-Z in shell, thus causing an exit. 
On next bootup I get:
record (xxx-xxx/.urb/egz.hope) is corrupt (j)

I tried to reproduce that on different comets but without luck. I guess some race condition was in effect. 

Any help and/or a new planet appreciated.

**Comment by cgyarvin** [comment]
Sorry!  I do know the maintenance event system has some race conditions.  What platform, though?

Galen will send you a new planet :-/

**Comment by mikolajpp** [comment]
No big deal :-)

Arch: Linux mercury 4.13.12-1-ARCH #1 SMP PREEMPT x86_64 GNU/Linux
Branch: v0.4.5

I tried to reproduce bug, but it is very hard. 
It seems it occurs more often with some additional apps running, 
but I haven't been able to pinpoint anything. 

One thing, it is definitely not related to pasting, because several times I managed to crash 
the comet just with ctrl-z. Definitely a race condition of sorts.
